### DISCUSSION TITLE
Reduce the size of AnkiMorphs

### DISCUSSION THREAD

**Discussion by aleksejrs** [mortii/anki-morphs#172] [General]
Every update is at least 30-40 MB.
Considering that deck downloading is limited on AnkiWeb, dae, is this important?
It also takes time.

**Comment by mortii** [comment]
Please don't ping Damien from random repositories he has never interacted with--it's very inconsiderate of his time and attention, and I imagine it's unlikely to be successful given that he probably has a trillion other notifications on/from github. Let him focus (with his very finite time) on things that are actually important and that he cares about.

> Every update is at least 30-40 MB.

The .addon files I upload to ankiweb are ~21 MB, so it would be pretty weird if your updates are 30MB+

[In this anki forums thread](https://forums.ankiweb.net/t/how-many-mb-is-the-limit-for-the-add-on-file-that-can-be-uploaded-to-ankweb/39069/2) he says the limit is approximately 150MB, so AM is well within the acceptable range from an ankiweb infrastructure point of view.

With that out of the way, your point about the add-on being too big is valid. For me it takes less than a second to download, but that will not be the case for everyone, and a size of 1KB would of course be much preferable to 21MB. 

ianki already made an add-on that only contains mecab which could then be imported to morphman and used as a morphemizer, and we could do the same thing in ankimorphs. It would break backwards compatibility, but it might be worth it in the long run.

**Comment by aleksejrs** [comment]
This is more convincing about not needing to ask Damien: https://forums.ankiweb.net/t/error-daily-limit-exceeded-please-try-again-tomorrow/34122

> When they’ve downloaded over 25 decks in a day.

**Reply by dae** [reply]
That limit is more about preventing people from downloading all the decks and then re-hosting them elsewhere (think knock-off apps).

**Comment by mortii** [comment]
I'll remove Mecab from AnkiMorphs in v2.0.0 (see issue #174).
